odoo/addons/test_lint/tests/_odoo_checker_unlink_override.py

```diff
@@ -0,0 +1,36 @@
+import astroid
+from pylint import checkers, interfaces
+
+
+class OdooBaseChecker(checkers.BaseChecker):
+    __implements__ = interfaces.IAstroidChecker
+    name = 'odoo'
+
+    msgs = {
+        'E8503': (
+            'Raise inside unlink override.',
+            'raise-unlink-override',
+            'Raising errors is not allowed inside unlink overrides, '
+            'you can create a method and decorate it with '
+            '@api.ondelete(at_uninstall=False), only use '
+            'at_uninstall=True if you know what you are doing.'
```

Comment:
Definitely requires explanation in the `@ondelete` docstring (in which situation would you want to flag one way or the other).